.github/CONTRIBUTING.md

```diff
@@ -43,18 +43,36 @@ You can (and should) run our test suite using [*tox*].
 However, you’ll probably want a more traditional environment as well.
 We highly recommend to develop using the latest Python release because we try to take advantage of modern features whenever possible.
 
-Clone the *attrs* repository to your computer:
+First [fork](https://github.com/python-attrs/attrs/fork) the repository on GitHub.
```

Comment:
```suggestion
First [fork](https://github.com/python-attrs/attrs/fork) the repository on GitHub.
It will only get a copy of the `main` branch unless you untick the following checkbox:
> * [x] Copy the `main` branch only
```